src/ui/components/Transfer.js

```diff
@@ -97,12 +106,37 @@ export const Transfer = ({
       });
       updateLedger(nextLedger);
       setAmount("");
+      setIsSavedToLedger(!isSavedToLedger);
```

Comment:
This is a bug. If you do 2 transfers, then isSavedToLedger would end up being false. This should be `setHasUnsavedChanges(true);`